WeakAuras/WeakAuras.lua

```diff
@@ -2605,6 +2639,61 @@ function WeakAuras.ResolveCollisions(onFinished)
   end
 end
 
+StaticPopupDialogs["WEAKAURAS_CONFIRM_REPAIR"] = {
+  text = "",
+  button1 = L["Run the repair tool"],
+  button2 = L["Cancel"],
+  OnAccept = function(self)
+     WeakAuras.RepairDatabase()
+  end,
+  OnShow = function(self)
+    if self.data.reason == "user" then
+      self.text:SetText(L["Manual Repair Confirmation Dialog"]:format(WeakAuras.LastUpgrade()))
+    else
+      self.text:SetText(L["Automatic Repair Confirmation Dialog"]:format(WeakAuras.LastUpgrade()))
+      self.button2:SetText(L["Continue Without Repairing"])
```

Comment:
The buttons didn't appear to resize with the text for me, this was overshot both sides.